tools/trigger-pipeline.ts

```diff
@@ -20,15 +20,17 @@ program
   .command('trigger <workflow-type>')
   .description(`trigger a workflow in the ci on the main branch`)
   .requiredOption('--api-token <api-token>', 'circleci api token')
-  .action(async (workflowType: string, options: { apiToken: string }) => {
-    const { apiToken } = options
+  .option('--branch <branch>', 'the branch the workflow will be triggered on', MAIN_BRANCH)
```

Comment:
what does this last parameter `MAIN_BRANCH` mean in this case?

Reply:
I think that is just the default if nothing is passed, see https://github.com/tj/commander.js#default-option-value